lookup/localpgp.go

```diff
@@ -29,7 +29,15 @@ type LocalPGPService struct {
 // NewLocalPGPService creates a new LocalPGPService if it finds a local
 // public keyring; otherwise it bails.
 func NewLocalPGPService() (*LocalPGPService, error) {
-	ringfile := path.Join(os.Getenv("HOME"), ".gnupg", "pubring.gpg")
+
+	home := os.Getenv("HOME")
+
+	// Check if an override for GNUPG home is set
+	if os.Getenv("GNUPGHOME") != "" {
+		home = os.Getenv("GNUPGHOME")
+	}
+
+	ringfile := path.Join(home, ".gnupg", "pubring.gpg")
```

Comment:
You are right, I could have tested this. I don't use $GNUPGHOME on the machine I coded this :)
I rebased my branch from master and changed the path. Should be correct now.